ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -2557,3 +2562,47 @@ def test_torch_instance_pow_(dtype_and_x, as_variable, native_array):
         class_name="tensor",
         method_name="pow_",
     )
+
+
+@st.composite
+def _get_dtype_and_multiplicative_matrices(draw):
+    return draw(
+        st.sampled_from(
+            [
+                _get_dtype_input_and_matrices(),
+                _get_dtype_and_3dbatch_matrices(),
+                _get_dtype_input_and_mat_vec(),
+            ]
+        )
+    )
+
+
+# matmul
+@handle_cmd_line_args
+@given(
+    dtype_tensor1_tensor2=_get_dtype_and_multiplicative_matrices().example(),
```

Comment:
`.example()` should not be called in the data generation. Perhaps you kept this in whilst testing?

Reply:
Hi @simonetgordon, yeah, my bad. I forgot to remove `.example()` from the test itself before committing the code. I have removed that, as well as `rtol` itself and tried fixing the tests. They run perfectly fine on my machine against **all** backends as of the latest commit. Looking forward to your feedback now.

Reply:
Hold up.

Someone refactored the entire testing criteria a few hours ago and made major changes to the frontend tests. I took the latest pull to merge any conflicts with my latest commits but when I try to run my tests, hypothesis fails throwing all sorts of weird errors. It isn't even able to collect the tests, running them is a different story altogether. I think I will wait out till these new changes are made stable enough and the errors are fixed on the `main` branch before running tests specific to just my function. 

@simonetgordon can you please have someone look into the `ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py` file so they can fix these errors as a consequence of the recent refactoring? Thanks!

Reply:
Hi @hmahmood24, yes I am aware of the recent frontend decorator changes, apologies for the inconvenience! I will let you know when they've been fixed.

Since you said the tests worked fine for you locally, hopefully we only need to wait for the decorator issues to be resolved to get this merged. If it takes too long, I will merge this PR within a reasonable amount of time.